yorm/tests/test_utilities.py

```diff
@@ -96,9 +110,86 @@ def it_requires_a_mapped_class_or_instance():
 
 def describe_match():
 
-    def it_is_not_yet_implemented():
-        with expect.raises(NotImplementedError):
-            utilities.match(Mock)
+    def class_factory(model_class, instance_pile):
+        matches = list(
+            utilities.match(
+                model_class,
+                (lambda kind, key: model_class(kind, key)),
```

Comment:
The lambda looks like it just takes two positional arguments and passes the same two positional arguments to the class constructer. I think this test would still pass if `factory` was `None`. What am I missing?